packages/sdk/README.md

```diff
@@ -17,6 +17,6 @@ We will announce the 1.0.0 release on Discord and on Twitter once it's ready, ke
 npm install @eth-optimism/sdk
 ```
 
-## API
+## Docs
```

Comment:
If the SDK is production ready, maybe update the above "NOTICE" section

Reply:
Still undergoing beta period. Will be ready for production in a few weeks, ideally.